src/login/login_auth.cpp

```diff
@@ -208,7 +208,20 @@ int32 login_parse(int32 fd)
         }
         break;
         case LOGIN_CREATE:
-            //looking for same login
+            Sql_EscapeString(SqlHandle, escaped_name, name.c_str());
+            Sql_EscapeString(SqlHandle, escaped_pass, password.c_str());
```

Comment:
[Looks like strings are already being escaped before the switch starts up at L103 and L104](https://github.com/project-topaz/topaz/blob/7e139a4645911acfca0712bd43a2a7a4efbe3041/src/login/login_auth.cpp#L102-L106).

Reply:
Totally was. The escape was already there so I didn't notice. Can be removed.

Fixed.